modules/n1ql/pages/n1ql-language-reference/alterindex.adoc

```diff
@@ -35,65 +39,70 @@ If a node goes down while an ALTER INDEX operation is in progress, then the inde
 Only users with the suitable RBAC role of `Administrator` are allowed to run the `ALTER INDEX` directive.
```

Comment:
Only users with the appropriate RBAC role of `Administrator` are ...
seems more correct?
Or _necessary_ role?

Reply:
Removed "suitable".